src/Adapters/Auth/mfa.js

```diff
@@ -87,11 +89,13 @@ class MFAAdapter extends AuthAdapter {
     }
     this.sms = validOptions.includes('SMS');
     this.totp = validOptions.includes('TOTP');
-    if (!this.sms && !this.totp) {
-      throw 'mfa.options must include SMS or TOTP';
+    this.email = validOptions.includes('EMAIL');
+    if (!this.sms && !this.totp && !this.email) {
+      throw 'mfa.options must include SMS or TOTP or EMAIL';
     }
     const digits = opts.digits || 6;
     const period = opts.period || 30;
```

Comment:
let's go with `period: number | Record<string, number>`. default is 30, it can be overriden for a specific factor by providing an object with overrides